README.md

```diff
@@ -1 +1,29 @@
-# subspace-cli
\ No newline at end of file
+# Subspace-CLI
+is a great tool that heavily simplifies the terminal workflow of the farming process. \\
+Instead of running a terminal instance for farmer, and running another terminal instance for the node, now you can run a SINGLE terminal instance to farm!
+
+## How to Use (commands)
+1. download the executable
```

Comment:
from where?

Reply:
@ozgunozerk ^ I think there's some workflows missing here?

Reply:
will update this link when we release the executable, should be downloaded from github